yal/change_preferences.py

```diff
@@ -633,8 +638,7 @@ async def state_update_bot_emoji_submit(self):
         choice = self.user.answers.get("state_update_bot_emoji")
         if choice == "skip":
             return await self.go_to_state("state_display_preferences")
-
-        data = {"persona_emoji": choice}
+        data = {"persona_emoji": extract_first_emoji(choice)}
```

Comment:
I don't think we were doing this before, but should we handle if the user doesn't send any emoji's? I guess we'll just use the default in that case, so maybe it doesn't matter?